### DISCUSSION TITLE
VRAM usage increase after update?

### DISCUSSION THREAD

**Discussion by wnslx** [comfyanonymous/ComfyUI#1314] [General]
Hi, my VRAM usage seems to have increased after updating. KSampler VRAM usage went from ~7 to ~10 and VAE Decode from ~9 to ~13. 

I've been using the program for maybe two weeks now. I'm using the portable version.

I can get back to lower usage by copying old \ComfyUI folder over the new one. And I can also update to xformers .21 and keep the lower VRAM usage with the old \ComfyUI folder.

Is this an intended feature, known issue or just me? Thanks.

**Comment by comfyanonymous** [comment]
Yes this is an intended feature. ComfyUI now tries to keep models in vram when you have some available to speed up overall gen times.

To disable this you can: --disable-smart-memory

**Comment by Kwisss** [comment]
> Yes this is an intended feature. ComfyUI now tries to keep models in vram when you have some available to speed up overall gen times.
> 
> To disable this you can: --disable-smart-memory

 --disable-smart-memory makes gen times significantly faster with more than 1 ksampler on my 1660 super with 6 gb vram
